src/Templates/form.twig

```diff
@@ -37,7 +37,7 @@
 {{ parent() }}
 {% include 'parts/table-footer.twig' %}
 <script type="text/javascript" src="{{baseurl}}js/vendor/jquery.validate.min.js"></script>
-<script type="text/javascript" src="{{baseurl}}js/vendor/tinymce.min.js"></script>
+<script src="//cdn.tinymce.com/4/tinymce.min.js"></script>
```

Comment:
In light of recent internet issues, do you think it's wise to use a CDN for this? Especially for just one, rather than the rest of them as well